modules/ROOT/pages/custom-ingress-configuration.adoc

```diff
@@ -106,14 +106,15 @@ Ingress controllers discover only those resources with a `spec.ingressClassName`
 +
 If you add the TLS section with a `secretName` attribute, you must ensure that you also create the referred TLS secret in the `rtf` namespace.
 
-<4> The `app-name` placeholder parameter will be replaced with the actual app name when you deploy the application. It helps to ensure that each endpoint name is unique if there is not a wildcard in the subdomain.
+<4> The `app-name` placeholder parameter is replaced with the actual app name when you deploy the application. It helps to ensure that each endpoint name is unique if there is not a wildcard in the subdomain.
 
 <5> A template can include multiple paths for a host, but Runtime Manager displays only the first path rule for the host.
 
-<6> These placeholder values are required for Kubernetes validation, but the actual values are not used by Runtime Fabric. 
+<6> The `path` parameter value is replaced by the value you add in the *Path* field when you xref:configure-ingress-for-a-mule-application[Configure Ingress for a Mule Application].
```

Comment:
Only clarification we need to explicitly add here is (if needed),
Note: `path` is not a placeholder. 
Reason: Users should not get confused and think they can use `path` as placeholder at other places in the template.

Reply:
ok, I'll add that here.